gradle-jdks/src/main/java/com/palantir/gradle/jdks/GradleWrapperPatcher.java

```diff
@@ -75,41 +87,51 @@ interface Params {
     public abstract static class GradleWrapperPatcherTask extends GradleWrapperPatcherTaskImpl {
 
         public GradleWrapperPatcherTask() {
-            onlyIf(t -> getGradleJdksSetupJar()
-                    .map(setupJar -> setupJar.getAsFile().exists())
-                    .getOrElse(false));
+            getGenerate().convention(false);
+            onlyIf(t -> getGradleJdksSetupJar().map(File::exists).getOrElse(false));
```

Comment:
Do we need this onlyIf? Can we ever end up running this task without extracting the gradle-jdks-setup jar first?

Reply:
Just worried it's going to silently do nothing rather than explode in an obvious way.

Reply:
> Do we need this onlyIf? Can we ever end up running this task without extracting the gradle-jdks-setup jar first?

You are right, this shouldn't happen.